packages/dds/tree/src/test/feature-libraries/modular-schema/comparison.spec.ts

```diff
@@ -303,7 +258,7 @@ describe("Schema Comparison", () => {
 			});
 
 			testOrder(compareTwoRepo, repos);
-			assert.equal(allowsRepoSuperset(defaultSchemaPolicy, repos[0], repos[1]), true);
```

Comment:
this check was entirely redundant with the above. The new check is not, as it includes comparing required and sequence (in both orders, and checks that they produce the correct results)